src/store/actions/appConfigStoreActions.ts

```diff
@@ -295,3 +295,12 @@ export const setNodeType = async (nodeType: NodeType) => {
         updateNodeTypeForNodeStore(nodeType);
     });
 };
+
+export const setWarmupSeens = (warmupSeen: boolean) => {
+    useConfigUIStore.setState({ warmup_seen: warmupSeen });
+    invoke('set_warmup_seen', { warmupSeen }).catch((e) => {
+        console.error('Could not set seen', e);
+        setError('Could not change seen');
+        useConfigUIStore.setState({ visual_mode: !warmupSeen });
```

Comment:
Shouldn't be a `warmup_seen` here? :imp:

Reply:
lol :') thank you!!